### DISCUSSION TITLE
How to use Svelte with Parcel

### DISCUSSION THREAD

**Discussion by rawhuul** [parcel-bundler/parcel#8883] [Q&A]
Since Svelte 3, it's my go to framework and I love parcel too. Can you guys help me in setting up both of them seemlessly.

**Comment by mischnic** [comment]
See https://github.com/parcel-bundler/parcel/issues/7503, the mentioned plugins might work for you

**Comment by rawhuul** [comment]
The transformer [@parcel-transformer-svelte3-plus](https://github.com/HellButcher/parcel-transformer-svelte3-plus) seems like unmaintained.
